cheroot/wsgi.py

```diff
@@ -276,6 +278,11 @@ def get_environ(self):
 
         return env
 
+    def _get_peer_uid(self):
```

Comment:
also done in 831ec2db35533bed15b2d9bf38894ccbaab2af00

I would be squashing that into the base commit as review is complete.